boards/posix/nrf52_bsim/Kconfig.board

```diff
@@ -10,6 +10,10 @@ config BOARD_NRF52_BSIM
 	select CONSOLE_HAS_DRIVER
 	select NRF_RTC_TIMER
 	select CLOCK_CONTROL
+	# Pretend that the nRF RNG peripheral is present, so that the driver
```

Comment:
Silly thing: You can remove the "Pretend that the nRF RNG peripheral is present , " part of this comment.
The RNG HW model is present :)

Reply:
Right. What about changing it to "Indicate that the nRF RNG peripheral (actually a model of it ) is present, ..."?

Reply:
Fine by me

Reply:
Changed.